Libraries/Components/TextInput/TextInput.js

```diff
@@ -71,10 +88,62 @@ type Event = Object;
  * `multiline=false`. To achieve the same effect, you can wrap your `TextInput`
  * in a `View`:
  *
- * ```
- *  <View style={{ borderBottomColor: '#000000', borderBottomWidth: 1, }}>
- *    <TextInput {...props} />
- *  </View>
+ * ```ReactNativeWebPlayer
+ * import React, { Component } from 'react';
+ * import { AppRegistry, View, TextInput } from 'react-native';
+ *
+ * class UselessTextInput extends Component {
+ *   constructor(props) {
+ *     super(props);
+ *     this.state = {
```

Comment:
Is this.state.text used anywhere in this component? It doesn't seem so. I'd get rid of the whole constructor if it isn't.